pallets/schemas/src/tests/mock.rs

```diff
@@ -35,19 +31,32 @@ frame_support::construct_runtime!(
 // See https://paritytech.github.io/substrate/master/pallet_collective/index.html for
 // the descriptions of these configs.
 
+const MILLISECS_PER_BLOCK: u64 = 12000;
```

Comment:
we probrably want to undo this

Reply:
It's already in a shared file. We are moving it back. We removed that dependency to help us do some debugging and get better compiler errors.